### PR TITLE
All News Page Styling

### PR DESCRIPTION
**Pull request by Karyum**
rel #114
screenshot from 2017-07-26 12-26-09
screenshot from 2017-07-26 12-26-20

### CODE REVIEW

File: src/views/all_news.hbs [hunk 1]
```diff
@@ -1,17 +1,17 @@
-<div >
-  <h1>Tech & Startup News</h1>
-  <div>
-    <div>
-      <a href="/news/{{_id}}"><img src="/assets/fac.png"></a>
-      <a href="/news/{{_id}}">{{this.SingleNews.newsTitle}}</a>
-      <p>{{this.news.subhead}}</p>
-      <a href="/news/{{_id}}">Read More</a>
+<div class="w-60 center db">
+  <h1 class="fl tracked mt4 mid-gray">Tech & Startup News</h1>
```
Comment (Suuuuha) [inline src/views/all_news.hbs]: I think we should keep on consistency with colors .. we have a 'text-granite-gray' in the css file which have been used for all the titles so far!

File: src/views/all_news.hbs [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+  <div class="dib w-100">
+    <div class="fl dib w-two-thirds mb2">
+      <a href="/news/{{this.SingleNews._id}}"><img src="/assets/fac.png" class="h-50  w-90 mv3 ba br2"></a><br>
+      <a href="/news/{{this.SingleNews._id}}" class="dib mid-gray no-underline f3 w-90">{{this.SingleNews.newsTitle}}</a>
```
Comment (Suuuuha) [inline src/views/all_news.hbs]: Same here

File: src/views/all_news.hbs (hunk 1 above)
Comment (Karyum) [inline src/views/all_news.hbs]: The thing is in the wireframe it looks darker and even if i used our color its not really readable do you think i should change it anyways?

Comment (Suuuuha) [inline src/views/all_news.hbs]: It looks fine for me , I dont mind changing it to the one you are using.. as long as we are both using the same one

Comment (Karyum) [inline src/views/all_news.hbs]: i did use text-granite-gray for the paragraphs but the title needs darker text thats why i used a tachyons color mid-gray

Comment (Suuuuha) [inline src/views/all_news.hbs]: ok Im approving then we'll see what @esraajb thinks and then we both will change to one of them :)

Comment (Karyum) [inline src/views/all_news.hbs]: @Suuuuha you dont need to change, keep what you have its just the news section needed a darker font

Comment (Suuuuha) [inline src/views/all_news.hbs]: The mid-gray is a bit too gray, the titles in the frameworks are more purple than gray

Comment (Karyum) [inline src/views/all_news.hbs]: im not sure i got the color from color picker and it said dark  gray which is pretty much mid gray

**Comment by Karyum** [comment]
@esraajb sorted

**Comment by esraajb** [comment]
@Karyum travis is failing

**Comment by esraajb** [comment]
lazy @Karyum

**Pull request merged**